website/client/js/services/achievementServices.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Services that handle achievement logic.
+ */
+
+angular.module('habitrpg').factory('Achievement',
+['$rootScope', function($rootScope) {
+
+  function displayAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl', size:'sm'});
+  }
+
+  function displayBulkyAchievement(achievementName) {
+    $rootScope.openModal('achievements/' + achievementName,
+      {controller:'UserCtrl'});
```

Comment:
Sorry, you're right that the default isn't `lg`. All size does is add an extra class to the modal `modal-${size}`. If you don't pass a size, it doesn't pass anything in. The equivalent size is `md`. So either you can pass `md` as the size or make no size the default and have the small achievements pass in `sm`

```js
function displayAchievement(achievementName, options) {
  options = options || {};

  $rootScope.openModal('achievements/' + achievementName, {
    controller: 'UserCtrl', 
    size: options.size // if undefined, will be md size
  });
}
```